tap_mysql/connection.py

```diff
@@ -89,7 +89,7 @@ def __init__(self, config):
             "host": config["host"],
             "port": int(config["port"]),
             "cursorclass": config.get("cursorclass") or pymysql.cursors.SSCursor,
-            "connect_timeout": CONNECT_TIMEOUT_SECONDS,
+            "connect_timeout": config.get("connect_timeout", DEFAULT_CONNECT_TIMEOUT_SECONDS) ,
```

Comment:
can you cast this to int? just in case `connect_timeout' is passed as stringified integer

Reply:
sure